map.html

```diff
@@ -73,9 +73,15 @@
 </div>
 <br/>
 
-<h1 style="text-align: center;">Map Area</h1>
-<p>This is where the explanation goes.</p>
-<div style="height:80vh;" id="wrapper_Map"></div>
+
+<div style="margin-top: 1em;">
+	<button id="button_Location" title="Shows all Locations that need to be discovered.">Locations</button> 
+	<button id="button_Nirnroot" title="Shows where all Nirnroot are in the overworld.">Nirnroots</button>
+	<button id="button_ToggleTSP" title="The Traveling Salesman Path(TSP) is the fastest way to visit each location/nirnroot">Toggle Traveling Salesman Path</button>
+	<button onclick="showHelp()">Help!</button>
+</div>
+
+<div style="height:85vh; border: solid 1px" id="wrapper_Map"></div>
```

Comment:
increased map height since we don't have as much space taken up by a header.

Reply:
hmm. ideally we would just auto format it to the rest of the window size. I'll see about doing a ui rewrite soon.